pkg/config/validation/validation_test.go

```diff
@@ -121,6 +121,35 @@ func TestValidateWildcardDomain(t *testing.T) {
 	}
 }
 
+func TestValidateTrustDomain(t *testing.T) {
+	tests := []struct {
+		name string
+		in   string
+		out  string
+	}{
+		{"empty", "", "empty"},
+		{"happy", strings.Repeat("x", 63), ""},
+		{"multi-segment", "foo.bar.com", ""},
+		{"middle dash", "f-oo.bar.com", ""},
+		{"trailing dot", "foo.bar.com.", ""},
+		{"prefix dash", "-foo.bar.com", "invalid"},
+		{"bad format", "foo/bar/com", "invalid"},
```

Comment:
these two name are the same? use different names?

Reply:
Will do.

Reply:
i see the updates. you can click "resolve" when it's been updated. so we'll know is resolved without looking.